addons/sms/models/sms.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.addons.iap.models import iap
+from odoo.addons.iap.models.iap import InsufficientCreditError
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import phonenumbers
+    _sms_phonenumbers_lib_imported = True
+
+except ImportError:
+    _sms_phonenumbers_lib_imported = False
```

Comment:
You can do phonenumbers=None, and remove  '_sms_phonenumbers_lib_imported' and replace the

if _sms_phonenumbers_lib_imported, by a if phonenumbers